Description: Add support for Debian GNU/Hurd.
 $^O is 'gnu' there.
Origin: vendor
Author: gregor herrmann <gregoa@debian.org>
Last-Update: 2016-08-13

--- a/lib/Clipboard.pm
+++ b/lib/Clipboard.pm
@@ -15,7 +15,7 @@
         # example, cygwin doesn't count as Unix here, because it will
         # use the Win32 clipboard.)
         bind_os(Xclip => qw(linux bsd$ aix bsdos dec_osf dgux
-            dynixptx hpux irix dragonfly machten next os2 sco_sv solaris sunos
+            dynixptx gnu hpux irix dragonfly machten next os2 sco_sv solaris sunos
             svr4 svr5 unicos unicosmk)),
         bind_os(MacPasteboard => qw(darwin)),
         bind_os(Win32 => qw(mswin ^win cygwin)),
